app/views/companies/show.html.haml

```diff
@@ -29,15 +29,18 @@
           %h2 #{t('.address')}:
           = field_or_none t('.street'), @company.main_address.street_address, tag_options: {class: 'street'}
 
+          = field_or_none t('.post_code'), @company.main_address.post_code, tag_options: {class: 'post-code'}
+
           = field_or_none t('.city'), @company.main_address.city, tag_options: {class: 'city'}
 
           = field_or_none t('.kommun'), @company.main_address.kommun.name, tag_options: {class: 'kommun'}
```

Comment:
2 question about this, @patmbolger :

1. what test data is failing?  Do you mean that some test are failing for you?  All tests pass for me and on Semaphore, so I'm confused.

2. Can you explain what `@company.main_address.kommun&.name` does?  What the `&` is doing?

I bet it's a good change to make, but I don't quite understand it.